src/test/java/core/basesyntax/service/FruitServiceTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Warehouse;
+import core.basesyntax.model.OperationFactory;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceTest {
+    private static final List<String[]> TEST_INPUT = new ArrayList<>();
+    private static final List<String[]> TEST_REPORT = new ArrayList<>();
+    private final FruitService fruitService =
+            new FruitService(new OperationFactory<>(Warehouse.getFruitStorage()));
+
+    @BeforeClass
+    public static void setUp() {
+        TEST_INPUT.add(new String[]{"b", "test", "10"});
+        TEST_REPORT.add(new String[]{"test", "10"});
+    }
+
+    @Test
+    public void importData_ok() {
+        fruitService.importData(TEST_INPUT);
+        assertTrue("One or more lines should be imported",
+                Warehouse.getFruitStorage().size() >= 1);
+    }
+
+    @Test
+    public void generateReport_ok() {
+        fruitService.importData(TEST_INPUT);
```

Comment:
put data directly to the storage instead of relying on the method that you are not testing at the moment